src/scripts/create_release.sh

```diff
@@ -5,6 +5,7 @@ set -eo pipefail
 
 function create_release() {
     missing_dependencies=false
+    orb_version="${ORB_VERSION:-1.1.2}"
```

Comment:
Where is this `ORB_VERSION` env var coming from? We cannot put it in the default executor because anecdotical feedback I had from customers is that they use their own executor. Also, I think the fallback should be something like `dev` or `unknown`, not a specific version ;-)

Reply:
Resolved in #958f886